.azuredevops/publish-helm-library.yaml

```diff
@@ -0,0 +1,79 @@
+name: $(BuildID)
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - 'main'
+  paths:
+    include:
+      - .azuredevops/*
```

Comment:
I think you can put * as you include everything.

Reply:
@ppotturi : Actually I have not included files outside above mentioned folders.

Reply:
@ppotturi : fixed